src/main/java/uk/gov/pay/ledger/transaction/model/CsvTransactionFactory.java

```diff
@@ -154,7 +156,9 @@ public CsvTransactionFactory(ObjectMapper objectMapper) {
         return result;
     }
 
-    public Map<String, Object> getCsvHeadersWithMedataKeys(List<String> metadataKeys, boolean includeFeeHeaders) {
+    public Map<String, Object> getCsvHeadersWithMedataKeys(List<String> metadataKeys,
+                                                           boolean includeFeeHeaders,
+                                                           boolean includeMotoHeader) {
```

Comment:
Adding another boolean is fine here but I think if we find ourselves adding any more we should refactor to use an `EnumSet` or something to represent which headers we want to include

Reply:
Yeh I agree